assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -101,7 +123,7 @@
                             <type>war</type>
                         </overlay>
                     </overlays>
-                    <packagingExcludes>WEB-INF/lib/che-core-ide-stacks-${che.version}.jar,
```

Comment:
Why don't we need exclusion of stacks anymore?

Reply:
Well, it seems it was mentioned twice in the `packagingExcludes` element. I just removed the duplicates.